packages/svelte-check/package.json

```
@@ -29,11 +29,11 @@
         "fast-glob": "^3.2.7",
         "import-fresh": "^3.2.1",
         "sade": "^1.7.4",
-        "svelte-preprocess": "^5.0.0",
-        "typescript": "^4.9.4"
+        "svelte-preprocess": "^5.0.0"
     },
     "peerDependencies": {
-        "svelte": "^3.55.0"
+        "svelte": "^3.55.0",
+        "typescript": "^4.9.0"
```

This should be updated to allow Typescript 5 to be used as well
```suggestion
        "typescript": ">=4.9.0"
```